packages/_debian-common/changelog

```diff
@@ -1,8 +1,12 @@
 libsuperiormysqlpp (0.4.1) UNRELEASED; urgency=medium
 
+  [ Peter Opatril ]
   * refactor: Logging system (DRY)
   * feat: Default logger logs errors
 
+  [ Michal Merc ]
+  * feat: Implement psQuery function
```

Comment:
I'd mention that you've also added ps{Param|Result}Query. You may of course just mention them as "convenience functions".